draft-ietf-jsonpath-base.md

```diff
@@ -1420,6 +1507,19 @@ substring that matches the iregexp contained in the string
 that is the second argument.
 The result is `true` if such a substring exists, `false` otherwise.
 
+### Examples
```

Comment:
this still uses "syntax", we probably want to say "typing" or "types"

Reply:
Fixed in 34b224404